trellis_server:
  ros__parameters:
    buffer_dir: /synthesis
    conversion_service: ~/generate
    conversion_model: /models/TRELLIS-image-large
    qsize: 1
